cairo1-run/Makefile

```diff
@@ -0,0 +1,2 @@
+make test:
+	cargo test
```

Comment:
We missed the newline. I wonder if we can configure the repo globally to insert them.